src/main/resources/META-INF/plexus/components.xml

```diff
@@ -17,6 +22,7 @@
        <package>org.apache.maven.plugins:maven-jar-plugin:2.6:jar,${project.groupId}:${project.artifactId}:${project.version}:build</package>
        <pre-integration-test>${project.groupId}:${project.artifactId}:${project.version}:start</pre-integration-test>
        <integration-test>org.apache.maven.plugins:maven-failsafe-plugin:2.19.1:integration-test</integration-test>
+       <!-- post-integration-test is opportunity for code coverage, such as jacoco, to collect statistics from running container -->
        <post-integration-test></post-integration-test>
        <verify>${project.groupId}:${project.artifactId}:${project.version}:stop,org.apache.maven.plugins:maven-failsafe-plugin:2.19.1:verify</verify>
```

Comment:
We use [jacoco:dump](http://eclemma.org/jacoco/trunk/doc/dump-mojo.html) to connect to port 6300.

Reviewing my personal usage of phases: mostly it's _mvn clean install_. For jenkins it's _mvn clean deploy_.  If I want to build without releasing to my local respository it's _mvn clean verify_.   I use _verify_ because I want to look at the jacoco and failsafe reports generated from the integration tests.  Occasionally while debugging an aspect of the build I might use _mvn package_.

I almost never use any of the really long named phases;  they take too long to type and I usually mis-remember the actual name.  I suspect that these phases are more about sequencing than about the expectation of use from the command line.  In the [Maven Introduction to the Build Lifecycle](http://maven.apache.org/guides/introduction/introduction-to-the-lifecycle.html#Build_Lifecycle_Basics), you don't see any of the _pre-*_, _post-*_, or _process-*_ phases mentioned until the reference section.